e2e/main_test.go

```diff
@@ -25,9 +24,6 @@ func TestMain(m *testing.M) {
 
 	log := zap.New(zap.UseFlagOptions(&zap.Options{
 		Development: true,
-		ZapOpts: []uberzap.Option{
-			uberzap.AddCaller(),
```

Comment:
I don't agree with this change. Please keep the file:lineno info. It might not seem useful when debugging but it is very useful for understanding the flow in the codebase through logs.

We should use log tools to remove unwanted fields when analyzing them. Skipping the fields while logging is not the right place.

Reply:
Can we get the file:line to be logged at the end? It makes following the logs harder.